library/psa_crypto_driver_wrappers.c

```diff
@@ -1995,4 +1995,14 @@ psa_status_t psa_driver_wrapper_free_random(
     return PSA_SUCCESS;
 }
 
+psa_status_t psa_driver_wrapper_destroy_key(const psa_key_attributes_t *attributes)
+{
+        psa_key_location_t location = PSA_KEY_LIFETIME_GET_LOCATION(attributes->core.lifetime);
+
+        switch (location) {
+        }
+
+        return PSA_ERROR_INVALID_ARGUMENT;
```

Comment:
```suggestion
        return PSA_ERROR_NOT_SUPPORTED;
```

Suggested change in error-code to be in sync with code [here](https://github.com/nrfconnect/sdk-nrf/blob/fc4927e4f50e4d45d9f7f2e85bcb94ac3a06454f/subsys/nrf_security/src/psa_crypto_driver_wrappers.c#L2835)